packages/components/src/Modal.tsx

```diff
@@ -94,6 +94,19 @@ const ModalOverlay = ({ isDismissable = true, ref, ...props }: ModalOverlayProps
 	return (
 		<AriaModalOverlay
 			isDismissable={isDismissable}
+			shouldCloseOnInteractOutside={(element) => {
```

Comment:
@cspath1 will this negatively interact with toolbar?

Also curious if this will have a negative impact on public consumers (who may be rendering any number of iframes etc)

Reply:
I don't think this change will negatively interact with the toolbar thankfully! As far as Modal usage within the toolbar is concerned, we aren't using the modal component (we have a popup window for the auth flow, since storing cookies via an iframe can cause issues).

More generally speaking though, I do think there is an _existing_ issue with any sort of auto-close functionality when it comes to interacting with the toolbar (i.e. you wouldn't be able to keep the modal open and also interact with the toolbar without the modal closing). This isn't too big of a deal, unless you want to use the toolbar specifically to update flag values/track events that occur within said modal.

I'm curious if there's anything we (FM Next) can do to mitigate that sort of issue on our end.